standalone/src/test/java/io/pravega/local/InProcPravegaClusterTest.java

```diff
@@ -56,6 +56,9 @@ public void setUp() throws Exception {
                                            .enableTls(tlsEnabled)
                                            .certFile("../config/cert.pem")
                                            .keyFile("../config/key.pem")
+                                           .jksKeyFile("../config/bookie.keystore.jks")
```

Comment:
What's the test plan for this feature? We need to have more test coverage. Also, shouldn't we be enabling the controller to SSL to ZK as well? Are you doing that separately?

Reply:
We are running the `standalone` tests and `integration` tests with ZK SSL enabled. Please note that the codecov coverage in this case is not reporting the diff properly.